src/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/ResourceSectionBuilder.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection.Metadata;
+
+namespace System.Reflection.PortableExecutable
+{
+    public abstract class ResourceSectionBuilder
+    {
+        protected ResourceSectionBuilder()
+        {
+        }
+
+        internal protected abstract void Serialize(BlobBuilder builder, SectionLocation location);
```

Comment:
ok

---
In reply to: [63404975](https://github.com/dotnet/corefx/pull/8419#discussion_r63404975) [](ancestors = 63404975)